### ISSUE TITLE
why threshold 0.05

### ISSUE THREAD

**Issue by ldz666666** [zylo117/Yet-Another-EfficientDet-Pytorch#483]
Thanks for your repository  
I wonder why in coco_eval,py line 55, the confidence threshold was set to 0.05 ,which seems too low for me and may bring many extra boxes

**Comment by alexriedel1** [comment]
Yeah you should have as many boxes as possible to effectively find all the true positives and false positives
